client/securedrop_client/sdk/__init__.py

```diff
@@ -120,14 +112,20 @@ def __init__(
         self.default_download_timeout = default_download_timeout or DEFAULT_DOWNLOAD_TIMEOUT
 
         self.proxy_vm_name = DEFAULT_PROXY_VM_NAME
-        config = configparser.ConfigParser()
+        config_parser = configparser.ConfigParser()
         try:
             if os.path.exists("/etc/sd-sdk.conf"):
-                config.read("/etc/sd-sdk.conf")
-                self.proxy_vm_name = config["proxy"]["name"]
+                config_parser.read("/etc/sd-sdk.conf")
+                self.proxy_vm_name = config_parser["proxy"]["name"]
         except Exception:
             pass  # We already have a default name
 
+        # Load download retry limit from the config
+        self.download_retry_limit: int = 3
+        config = Config.load()
+        if config.download_retry_limit is not None:
+            self.download_retry_limit = int(config.download_retry_limit)
```

Comment:
Good point, @micahflee, re: typing.  Should `Config.load()` validate and enforce typing at the `dataclass` level?